src/NServiceBus.Transport.RabbitMQ.Tests/NServiceBus.Transport.RabbitMQ.Tests.csproj

```
@@ -7,11 +7,22 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="myp12.p12" />
```

Assuming the path change for the file stays in the PR, then this should be updated as well:

```suggestion
    <None Update="myp12.p12" CopyToOutputDirectory="PreserveNewest" />
```